tests/php/Forms/GridField/GridFieldFilterHeaderTest.php

```diff
@@ -208,8 +208,8 @@ public function testCanFilterAnyColumns()
 
         // test that you cannot filter by non-db field when it falls back to summary_fields
         Config::modify()->remove(Team::class, 'searchable_fields');
-        Config::modify()->set(Team::class, 'summary_fields', ['MySummaryField']);
-        $this->assertFalse($filterHeader->canFilterAnyColumns($gridField));
+        Config::modify()->set(Team::class, 'summary_fields', ['Cheerleader.Name']);
+        $this->assertTrue($filterHeader->canFilterAnyColumns($gridField));
```

Comment:
Why have you made this change? The comment above this makes it clear that the intention is to test that you _cannot_ filter by non-db fields... but here you're testing that you _can_ filter by _db fields_